flink-examples/flink-examples-streaming/pom.xml

```diff
@@ -168,20 +162,6 @@ under the License.
 						<goals>
 							<goal>unpack</goal>
 						</goals>
-						<configuration>
```

Comment:
can we not remove the entire plugin?

Reply:
What was this doing in the first place?...

Reply:
hmm, doesn't seem to do anything. The twitter jar doesn't contain anything in the included directory after all.